targetplatform/smarthome.target

```diff
@@ -55,6 +55,7 @@
 <unit id="org.codehaus.jackson.mapper.source" version="1.6.0.v20101005-0925"/>
 <unit id="org.hamcrest.core" version="1.3.0.v201303031735"/>
 <unit id="org.hamcrest.core.source" version="1.3.0.v201303031735"/>
+<unit id="org.hamcrest" version="1.1.0.v20090501071000"/>
```

Comment:
When doing changes to this file, you need to increase the sequenceNumber in line 2 above.